docs/node/tutorial.mdx

```diff
@@ -151,6 +152,58 @@ Current validator set:
   1. {pubkey = a2d8e307117695c8d0d3adbda136cb2ad630c43b8672a0eba46d79501831b794, power = 1}
 ```
 
+## Deploying A Database
+
+To deploy a database to the network, we will need to locate the network chain ID and any of the node's HTTP listen addresses.
+
+### Getting The Chain ID
```

Comment:
I started looking at this page and I was wondering why we are running `kwild` with a non-default `app.admin-listen-addr`.  The default allows `kwild` and `kwil-cli` to Just Work without specifying an `rpcserver`.

I think the reason is that we are writing the tutorial to demonstrate some basic tasks (start a new network, add a second node, etc.) without actually having a second machine to run that second node?  I guess that's reasonable and preferable to changing the instructions to read like "And then on the other machine where the second node will run..."
If so I guess we can run with this, but it strikes me as overly contrived and does not reflect an actual deployed network.  Which of our docs pages would be best to describe a realistic example, where a new network of nodes on different machines is created?

Reply:
I think the reason we run both nodes on a non-standard `app.admin-listen-addr` is because we asume that the person running this tutorial is running both nodes on the same machine. Therefore, if both nodes use the default `app.admin-listen-addr` (which I think is `unix:///tmp/kwil_admin.sock`), then you won't be able to send validator join / approve requests, because which node are the requests actually coming from.

If the tutorial assumes you are running nodes on separate machines, then we can use the default `app.admin-listen-addr`, but I think that would be a worse tutorial because then users have to go through the trouble of using multiple machines (which is non-trivial if you don't have much DevOps experience). IMO, this tutorial should allow anyone to just copy the bash commands we provide, paste them in a terminal, and be able to conceptually understand what Kwil can do.